client/src/pages/StudyPost.tsx

```diff
@@ -0,0 +1,197 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useNavigate, Link } from "react-router-dom";
+import axios from "axios";
+
+import TextEditor from "../components/TextEditor";
+
+const StudyPost = () => {
+  const [postText, setPostText] = useState<string>("");
+  const [maxPeople, setMaxPeople] = useState<number>(0);
+  const [platform, setPlatform] = useState<string>("");
+
+  const navigate = useNavigate();
+
+  const handleMaxPeople = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setMaxPeople(+e.target.value);
+  };
+  const handlePlatform = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPlatform(e.target.value);
+  };
+
+  const handlePostButton = () => {
+    // console.log("Submitting form with values:", {
+    //   maxClassmateCount: maxPeople,
+    //   platform: `${platform}`,
+    //   introduction: `${postText}`,
+    // });
+
+    axios
+      .post(`${import.meta.env.VITE_APP_API_URL}/studygroup/1`, {
+        maxClassmateCount: maxPeople,
+        platform: `${platform}`,
+        introduction: `${postText}`,
+      })
+      .then(() => alert("스터디 등록이 완료되었습니다!"))
```

Comment:
.then()을 체이닝 하기보다 await로 바꿔보세요!
```suggestion
      .then(() => alert("스터디 등록이 완료되었습니다!"))
      const {data, status} = await axios.post(...)
```